ruby/private/binary_wrapper.tpl

```diff
@@ -122,28 +118,10 @@ def main(args)
       end
     end
   end
-  exec(ruby_program, '--disable-gems', *rubyopt, main, *args)
+  exec(ruby_program, *rubyopt, main, *args)
```

Comment:
?

Reply:
Don't we need access to RubyGems subsystem when running our ruby code?

Reply:
this was specifically a problem with rubocop which requires the ability to inspect the gems it depends on, e.g.  https://github.com/rubocop-hq/rubocop/blob/master/lib/rubocop/yaml_duplication_checker.rb